subsystem_controllers/config/drivers_controller_config.yaml

```diff
@@ -2,37 +2,34 @@
   ros__parameters:
     # Long: Timeout for each service to be detected as available in milliseconds
     # Units: milliseconds
-    service_timeout_ms : 200
+    service_timeout_ms: 200
 
     # Long: Timeout for each service call in milliseconds
     # Units: milliseconds
-    call_timeout_ms : 1000
+    call_timeout_ms: 1000
 
     # String: The namespace for nodes in this subsystem. All nodes under this namespace will have their lifecycle's managed by this controller
     subsystem_namespace: /hardware_interface
 
     # Required subsystem controller nodes for the overall system to be functional and will have their lifecycle's managed
-    required_subsystem_nodes: ['']
+    required_subsystem_nodes: [""]
 
-    # List of nodes which will not be directly managed by this subsystem controller 
+    # List of nodes which will not be directly managed by this subsystem controller
     # but which are required to be operational for the subsystem to function
-    unmanaged_required_nodes: ['']
+    unmanaged_required_nodes: [""]
 
     # List of nodes in the namespace which will not be managed by this subsystem controller
     # Specifically includes the lidar and gps nodes which are handled in other subsystem controllers
-    excluded_namespace_nodes : ['']
-    
-    # List of ros1 controller drivers (node name) to consider required and who's failure shall result in automation abort
-    ros1_required_drivers: ['']
-
-    # List of ros1 camera drivers (node name) to consider required and who's failure shall result in automation abort.
-    ros1_camera_drivers: ['']
-    
+    excluded_namespace_nodes: [""]
+
+    # ros1 controller driver (node name) to consider required and whose failure shall result in automation abort
+    ros1_ssc_driver_name: ""
+
     # Boolean: If this flag is true then all nodes under subsystem_namespace are treated as required in addition to any nodes in required_subsystem_nodes
     full_subsystem_required: false
 
     # Int: The time allocated for system startup in seconds
     startup_duration: 30
 
     # Double: The timeout threshold for essential drivers in ms
-    required_driver_timeout: 3000.0
\ No newline at end of file
+    required_driver_timeout: 3000.0
```

Comment:
This can be renamed to ssc_driver_timeout for clarity

Reply:
renamed